app/models/labware_creators/tagged_plate.rb

```diff
@@ -79,6 +79,10 @@ def convert_tag_plate_to_new_purpose
 
     def create_labware!
       create_plate! do |plate_uuid|
+        # TODO: {Y24-190} Work out a way to call the `create!` method on TagLayoutTemplate model in Sequencescape
+        #       using the V2 API. I think either we need to misuse the PATCH method with some kind of
+        #       attributes telling Sequencescape to run the `create!` method, or we need to create a new
+        #       endpoint associated with a TagLayoutTemplate that will run the `create!` method.
```

Comment:
I thought you were planning to redesign it so that you could call `create!` on `TagLayout` instead, and that pulled info from the `TagLayoutTemplate`?

Reply:
Yes, but we don't have `TagLayout` in V2 yet.  I plan to do that in a soon pull request.  This comment was written before we came across that solution.  I'm not sure it's worth updating since it's coming out in the next week.